{"uuid": "ae7c8de5-8a13-4a1e-a738-204af637f88a", "children": ["7812f6d0-601e-4207-b7ee-ae86431c68a9"], "befores": [{"name": "back_fixture", "status": "passed", "start": 1652795941654, "stop": 1652795941654}], "afters": [{"name": "back_fixture::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"D:\\xiaxy\\python\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"D:\\xiaxy\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 941, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\xiaxy\\automation_test2022.4.22\\testcases\\conftest.py\", line 74, in back_fixture\n    com.back()\n  File \"D:\\xiaxy\\automation_test2022.4.22\\common\\common_fun.py\", line 367, in back\n    self.get_clickable_element(self.Back,\"返回\")\n  File \"D:\\xiaxy\\automation_test2022.4.22\\common\\common_fun.py\", line 276, in get_clickable_element\n    raise e\n  File \"D:\\xiaxy\\automation_test2022.4.22\\common\\common_fun.py\", line 270, in get_clickable_element\n    ele = WebDriverWait(self.driver,timeout,0.5).until(EC.element_to_be_clickable(loc))\n  File \"D:\\xiaxy\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1652795952860, "stop": 1652795964381}], "start": 1652795941654, "stop": 1652795964381}